src/dialogue/mod.rs

```diff
@@ -140,6 +171,24 @@ where
     }
 }
 
+/// Result of dialogue handler
+#[derive(Debug)]
+pub enum DialogueResult<S> {
+    /// Next state
+    Next(S),
+    /// Exit from dialogue
+    Exit,
```

Comment:
We have `DialogueState` and can add `is_exit()` method

Reply:
So we have to add Exit => Exit boilerplate